### DISCUSSION TITLE
Bike-lane adjacent curb zone

### DISCUSSION THREAD

**Discussion by zanebclark** [openmobilityfoundation/curb-data-specification#105] [Data Specification]
Do we plan on supporting a "bike-lane adjacent" field for [Curb Zone](https://github.com/openmobilityfoundation/curb-data-specification/tree/main/curbs#curb-zone)? If someone were to double park next to a bike lane, the safety concerns would be greater and might be worth quantifying separately.

**Comment by schnuerle** [comment]
This could be supported. How do think it could be integrated into the current CDS objects?

**Reply by cartograham** [reply]
> This could be supported. How do think it could be integrated into the current CDS objects?

I am generally in favor of capturing presence/absence of roadway facilities like this as simple Boolean values. If more substantially detailed roadway inventory attributes are desired, I’d recommend pointing to the roadway system of record which should be accessible via linear reference or a key/segment ID.
